Configuration/TypoScript/constants.txt

```diff
@@ -70,6 +70,9 @@ plugin.tx_dpf {
         # cat=plugin.tx_dpf/development; type=boolean; label=Activates the the process number column in researcher workspace/search lists.
         showProcessnumberColumnInResearcherWorkspaceLists = 1
 
+        # cat=plugin.tx_dpf/activateAlwaysSetDateIssued; type=boolean; label=Activates the generation of "date issued" only for documents with a fulltext.
```

Comment:
The parameter name and the parameter description are contradicting. Do we "always" set the dateIssued as the name implies, or only when there is fulltext as the description says? From the code it's the former. So I think the documentation should be adjusted.

Reply:
I've adjusted the description to better describe the actual meaning of the configuration parameter.